.circleci/images/primary/README.md

```diff
@@ -0,0 +1,40 @@
+# How to update
+
+## Edit and build new image
+
+```
+cd .circleci/images/primary
+<make your changes>
+docker build -t maltoe/chromic-pdf-primary:x.y.z .
+```
+
+## Test the image
+
+By method of your choice. Disable the default seccomp profile. This will work for example:
+
+```
+docker run -it -v $(pwd):/src --rm --security-opt seccomp=unconfined maltoe/chromic-pdf-primary:x.y.z
+```
+
+Inside container:
+
+```
+# there are other means of getting files into the container, but this is simple enough
+$ cp -r /src .
+$ cd src
+$ mix deps.get
+$ MIX_ENV=integration mix test
+```
+
+## Push new image to docker hub
+
+First acquire the credentials from @maltoe.
+
+```
+docker login
+docker push maltoe/chromic-pdf-primary:x.y.z
+```
+
+## Update the circleci config
+
+in `.circleci/config.yml`
```

Comment:
Maybe https://circleci.com/blog/using-circleci-workflows-to-replicate-docker-hub-automated-builds/ ?

Reply:
you mean to test that the image still builds? :shrug: don't think that's necessary

would be nice if one could somehow build and store it in some circleci repo, to stop circleci from downloading the image every time

Reply:
no I meant autobuilding if the image changes and using latest